pkg/stats/statsmgr.go

```diff
@@ -92,3 +139,27 @@ func (s *StatsMgr) startWorker(numReadingFiles int) {
 		}
 	}
 }
+
+func countLines(path string) (int64, error) {
```

Comment:
Should these be considered?
https://github.com/vesoft-inc/nebula-importer/blob/master/pkg/config/config.go#L87-L89

Reply:
@jievince How do you think?